docs/src/pages/guides/usage.astro

```diff
@@ -431,7 +431,7 @@ import Link from "../../components/Link.astro";
     is open or closed. Additionally, it might be helpful to communicate when a range
     selection has started and ended via a polite <code>aria-live</code> region. This
     could be achieved with the <code>rangestart</code> and <code>rangeend</code>
-    <a href="http://localhost:4321/cally/components/calendar-range/#events"
+    <a href="/components/calendar-range/#events"
```

Comment:
I think this is incorrect, it could need the `/cally` prefix. But I have already written a component to make sure links are correctly handled. Can you swap to:

```suggestion
    <Link href="/components/calendar-range/#events"
```

This should do the right thing 👍